plugins/foreman_discovery/nightly/index.md

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
Isn't this needed to mark it as metadata?

Reply:
Good catch that was an accident. Put back.